pkg/pvcresizing/pvc.go

```diff
@@ -21,3 +29,12 @@ func PrometheusVolumeSize(annotationValue string) string {
 		return "100Gi"
 	}
 }
+
+func GetRetentionSize(storageSize resource.Quantity) string {
+	// Set Retention.Size (TSDB limit) to a ratio of the volume storage size.
+	storageSize.Set(int64(storageSize.AsApproximateFloat64() * key.PrometheusVolumeStorageLimitRatio))
+	// Trick to fit with the expected format in RetentionSize property
```

Comment:
just a small remark...  
Better to document this above the function, because this is a helper function.